src/components/Course/Course.vue

```diff
@@ -94,6 +123,19 @@ export default {
               error.toString())
           })
   },
+  methods: {
+    getId(id) {
+      this.course_id = id
+    },
+    deleteCourse() {
+      CourseService.deleteCourse(this.course_id)
+        .then((response) => {
+          console.log(response.data.message)
```

Comment:
Should be an alert, not a console log

Reply:
Mala onda! You must accept my pull request! :'(

Reply:
Ya subí los cambios del SweetAlert, pero no me acuerdo si el pull se agrega solo.